docs/source/tutorials/configuration-and-setup.rst

```diff
@@ -0,0 +1,113 @@
+==============================
+Configuration and plugin setup
+==============================
+
+Maybe you :doc:`played with commands </tutorials/playing-command>` for your
+plugin and now you want to make your plugin configurable. If you run an
+instance of Sopel yourself, you probably had to open and edit its
+:doc:`configuration</configuration>` file.
+
+Usually located in your home directory, under the ``.sopel/`` folder, the
```

Comment:
```suggestion
Usually located in the ``.sopel/`` folder under your home directory, the
```

I _think_ this is a bit clearer, without going all the way to "`~/.sopel/` folder".